tests/semanticdb/metac.expect

```diff
@@ -366,115 +375,115 @@ Symbols => 109 entries
+classes/C4#copy(). => method copy (param x: Int): C4
+classes/C4#copy().(x) => param x: Int
+classes/C4#x. => val method x Int
+classes/C4. => final object C4 extends Object { self: <?>.C4.type => +4 decls }
+classes/C4.apply(). => method apply (param x: Int): C4
+classes/C4.apply().(x) => param x: Int
+classes/C4.toString(). => method toString => String
+classes/C4.unapply(). => method unapply (param x$1: C4): C4
+classes/C4.unapply().(x$1) => param x$1: C4
+classes/C6# => case class C6 extends Object with Product with Serializable { self: C6 => +5 decls }
+classes/C6#_1(). => method _1 => Int
+classes/C6#`<init>`(). => primary ctor <init> (param x: Int): C6
+classes/C6#`<init>`().(x) => param x: Int
+classes/C6#copy$default$1(). => method copy$default$1 => Int @uncheckedVariance
+classes/C6#copy(). => method copy (param x: Int): C6
+classes/C6#copy().(x) => param x: Int
+classes/C6#x. => val method x Int
+classes/C6. => final object C6 extends Object { self: <?>.C6.type => +4 decls }
+classes/C6.apply(). => method apply (param x: Int): C6
+classes/C6.apply().(x) => param x: Int
+classes/C6.toString(). => method toString => String
+classes/C6.unapply(). => method unapply (param x$1: C6): C6
+classes/C6.unapply().(x$1) => param x$1: C6
+classes/C7# => class C7 extends Object { self: C7 => +2 decls }
+classes/C7#`<init>`(). => primary ctor <init> (param x: Int): C7
+classes/C7#`<init>`().(x) => param x: Int
+classes/C7#x. => val method x Int
+classes/C8# => class C8 extends Object { self: C8 => +2 decls }
+classes/C8#`<init>`(). => primary ctor <init> (param x: Int): C8
+classes/C8#`<init>`().(x) => param x: Int
+classes/C8#x. => val method x Int
+classes/C9# => class C9 extends Object { self: C9 => +2 decls }
+classes/C9#`<init>`(). => primary ctor <init> (param x: Int): C9
+classes/C9#`<init>`().(x) => param x: Int
+classes/C9#x(). => var method x Int
+classes/C10# => class C10 extends Object { self: C10 => +2 decls }
+classes/C10#`<init>`(). => primary ctor <init> (param s: => String): C10
+classes/C10#`<init>`().(s) => param s: => String
+classes/C10#s. => val method s => String
+classes/C11# => class C11 extends Object { self: C11 => +2 decls }
+classes/C11#`<init>`(). => primary ctor <init> (): C11
+classes/C11#foo(). => macro foo => Int
+classes/C11#foo(). => macro foo => Int
+classes/C12# => class C12 extends Object { self: C12 => +8 decls }
+classes/C12#Context# => class Context extends Object { self: Context => +2 decls }
+classes/C12#Context#Expr# => type Expr [typeparam T ]
+classes/C12#Context#Expr#[T] => typeparam T 
+classes/C12#Context#`<init>`(). => primary ctor <init> (): Context
+classes/C12#`<init>`(). => primary ctor <init> (): C12
+classes/C12#foo1(). => macro foo1 (param x: Int): Int
+classes/C12#foo1(). => method foo1 (param x: Int): Int
+classes/C12#foo1().(x) => param x: Int
+classes/C12#foo1().(x) => param x: Int
+classes/C12#foo1Impl(). => method foo1Impl (param context: Context)(param x: <?>.context.Expr[Int]): <?>
```

Comment:
the return type is empty here, something to do with curried functions maybe?

Reply:
You are right! fixed https://github.com/lampepfl/dotty/pull/12885/commits/139491bf42806215007d785d7995ae9e6615b402